fury/actor.py

```diff
@@ -3451,7 +3465,24 @@ def _textured_sphere_source(theta=60, phi=60):
 
 
 def texture_on_sphere(rgb, theta=60, phi=60, interpolate=True):
+    """
+    Map an RGB or RGBA texture on a sphere.
```

Comment:
same comment as above:

- `Use` instead of `Uses`   -> Imperative mood everywhere.
- Dot at the end of the sentence
- Also, it should be a one-liner. See numpy standard that we follow: https://numpydoc.readthedocs.io/en/latest/format.html